workalendar/europe/austria.py

```diff
@@ -1,9 +1,12 @@
 # -*- coding: utf-8 -*-
+from __future__ import unicode_literals
 from workalendar.core import WesternCalendar, ChristianMixin
+from workalendar.registry import iso_register
 
 
+@iso_register('AT')
 class Austria(WesternCalendar, ChristianMixin):
-    "Austria"
+    name = 'Austria'
```

Comment:
Do you mind creating a name property that return the docstring rather than changing all calendar to remove the docstring and adding a name property?

```
@property
def name(self):
    return self.__doc__
```

Reply:
I think class variable is a better place for things you want to use in your code rather than calling `__doc__`. Docstrings are for comments and docs. Besides, add a character and everything breaks.

Reply:
For me name is needed as I have helper class implemented in my Django app to display translated form choices and I use name attribute for translations. I could do without name, only with ISO code, but I think it's good idea to have such a variable on every Calendar instance.